src/interfaces/taskDetails.type.ts

```diff
@@ -29,6 +29,9 @@ export type DependencyListProps = {
 export type TaskDetailsProps = {
     detailType: string;
     value?: string;
+    icon_url?: string;
```

Comment:
- if the url being used here is a url of task details should we rename this to `url`?

```suggestion
    url?: string;
```